### DISCUSSION TITLE
how to make Brick extra shingles??

### DISCUSSION THREAD

**Discussion by firestorm2039** [ldtteam/minecolonies#8448] [Q&A] [answered]
just building the townhall and its asking me for brick extra shingles, i have JEI mod and i cant find it on there? there is clay shingles which as LOADS of different options, but cant see brick extra shingles?, i believe these used to be called clay oak shingles? but that recipe doesn't work now

**Comment by MotionlessTrain** [comment, marked as answer]
You need to use the architect's cutter for that.
This [wiki page](https://wiki.minecolonies.ldtteam.com/source/dependencies/domumornamentum) has more information about that.

Please use our discord server for further questions: discord.minecolonies.com
It is easier for us to answer questions there

**Reply by 4everLink** [reply]
Although the wiki does give some details about the architects cutter, it gives no information specifically about the "brick extra" material or how to make it. without that information, it's impossible to craft the "brick extra shingles" that a town hall requires. and the recipe for "brick extra" has not appeared in the recipe book, so currently the only hope is to accidentally stumble upon it.

**Reply by uecasm** [reply]
JEI is your friend.  Always always install that when playing with mods.

**Comment by atomicpapa** [comment]
In case someone ends up here looking for an answer like I did, here's a link to the "Extra" recipes on the wiki

https://wiki.minecolonies.ldtteam.com/source/items/extrablocks
